samples/matter/lock/src/credentials/credentials_data_types.h

```diff
@@ -118,6 +119,196 @@ struct Credential {
 	CHIP_ERROR Deserialize(const void *buff, size_t buffSize);
 };
 
+struct WeekDaySchedule {
+	union Data {
+		struct Fields {
+			uint8_t mDaysMask;
+			uint8_t mStartHour;
+			uint8_t mStartMinute;
+			uint8_t mEndHour;
+			uint8_t mEndMinute;
+		} mFields;
+		uint8_t mRaw[sizeof(mFields)];
+	};
+	static_assert(sizeof(Data) == 5);
```

Comment:
Hmm without `packed` I'd expect the resulting size to be 8.

Reply:
Static assertions didn't point to failure.

Reply:
Ahh ok, probably because we have only 1 byte fields here.